packages/generate-persisted-query-manifest/cli.js

```diff
@@ -65,7 +66,22 @@ program
     );
     writeFileSync(outputPath, JSON.stringify(manifest, null, 2));
 
-    console.log(`Manifest written to ${outputPath}`);
+    if (manifest.operations.length === 0) {
+      console.warn(
+        chalk.yellow(
+          "Warning: no operations found during manifest generation. " +
+            "You may need to adjust the glob pattern used to search " +
+            "files in this project. See the README for more information on how to configure the glob pattern: " +
+            "https://www.npmjs.com/package/@apollo/generate-persisted-query-manifest\n",
+        ),
+      );
+    }
+
+    console.log(
+      `${chalk.green("✓")} Manifest written to ${outputPath} with ${
+        manifest.operations.length
+      } operations.`,
```

Comment:
Special case 1?

Reply:
🤦‍♂️ n00b moment. Thanks!

Reply:
Fixed in [d5b9bb6](https://github.com/apollographql/apollo-utils/pull/348/commits/d5b9bb6ff051627e514e95fa9c4bd3527903748c)